README.md

```diff
@@ -171,66 +176,186 @@ Follow the guide https://supabase.io/docs/guides/auth#third-party-logins
-      </intent-filter>
-    </activity>
-  </application>
-</manifest>
-```
+  <plist>
+  <dict>
+    <!-- ... other tags -->
+    <key>CFBundleURLTypes</key>
+    <array>
+      <dict>
+        <key>CFBundleTypeRole</key>
+        <string>Editor</string>
+        <key>CFBundleURLSchemes</key>
+        <array>
+          <string>[YOUR_SCHEME]</string>
+        </array>
+      </dict>
+    </array>
+    <!-- ... other tags -->
+  </dict>
+  </plist>
+  ```
+
+  This allows for your app to be started from `YOUR_SCHEME://ANYTHING` links.
+
+  For more info: https://developer.apple.com/documentation/xcode/defining-a-custom-url-scheme-for-your-app
+
+</details>
+
+## Windows
+
+<details>
+  <summary>How to setup</summary>
+
+  Setting up deeplinks in Windows has few more steps than other plantforms. 
+
+  Declare this method in <PROJECT_DIR>\windows\runner\win32_window.h
+  ```cpp
+    // Dispatches link if any.
+    // This method enables our app to be with a single instance too.
+    // This is optional but mandatory if you want to catch further links in same app.
+    bool SendAppLinkToInstance(const std::wstring& title);
+  ```
+
+  Add this inclusion at the top of <PROJECT_DIR>\windows\runner\win32_window.cpp
+  ```cpp
+  #include "app_links_windows/app_links_windows_plugin.h"
+  ```
+
+  Add this method in <PROJECT_DIR>\windows\runner\win32_window.cpp
+  ```cpp
+  bool Win32Window::SendAppLinkToInstance(const std::wstring& title) {
+    // Find our exact window
+    HWND hwnd = ::FindWindow(kWindowClassName, title.c_str());
+    
+    if (hwnd) {
+      // Dispatch new link to current window
+      SendAppLink(hwnd);
+
+      // (Optional) Restore our window to front in same state
+      WINDOWPLACEMENT place = { sizeof(WINDOWPLACEMENT) };
+      GetWindowPlacement(hwnd, &place);
+      switch(place.showCmd) {
+        case SW_SHOWMAXIMIZED:
+            ShowWindow(hwnd, SW_SHOWMAXIMIZED);
+            break;
+        case SW_SHOWMINIMIZED:
+            ShowWindow(hwnd, SW_RESTORE);
+            break;
+        default:
+            ShowWindow(hwnd, SW_NORMAL);
+            break;
+      }
+      SetWindowPos(0, HWND_TOP, 0, 0, 0, 0, SWP_SHOWWINDOW | SWP_NOSIZE | SWP_NOMOVE);
+      SetForegroundWindow(hwnd);
+      // END Restore
+
+      // Window has been found, don't create another one.
+      return true;
+    }
+
+    return false;
+  }
+  ```
+
+  Add the call to the previous method in `CreateAndShow`
+  ```cpp
+  bool Win32Window::CreateAndShow(const std::wstring& title,
+                                  const Point& origin,
+                                  const Size& size) {
+  if (SendAppLinkToInstance(title)) {
+      return false;
+  }
 
-The `android:host` attribute is optional for Deep Links.
+  ...
+  ```
 
-For more info: https://developer.android.com/training/app-links/deep-linking
+  At this point, you can register your own scheme.  
+  On Windows, URL protocols are setup in the Windows registry.
 
-### For iOS
```

Comment:
Why was "For iOS" removed?

Reply:
Sorry, it is just moved slightly. 
https://github.com/supabase-community/supabase-flutter/pull/136/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R211